### PR TITLE
Service un/installation

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

File: src/fiskaltrust.Launcher/Extensions/LoggerConfigurationExtensions.cs [hunk 1]
```diff
@@ -19,16 +19,20 @@ public static LoggerConfiguration AddLoggingConfiguration(this LoggerConfigurati
...
+            .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}{EnrichedPackage}{EnrichedId}{EnrichedContext}] {Message:lj}{NewLine}{Exception}")
             .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
             .MinimumLevel.Override("System", LogEventLevel.Warning)
             .MinimumLevel.Override("Grpc", LogEventLevel.Warning)
             .MinimumLevel.Override("ProtoBuf", LogEventLevel.Warning)
             .WriteTo.Logger(configuration =>
                 configuration
                     .Filter.ByExcluding(Matching.WithProperty("EnrichedId"))
-                    .WriteTo.File(Path.Join(launcherConfiguration.LogFolder, $"log{suffix}-.txt"), rollingInterval: RollingInterval.Day, shared: true));
+                    .WriteTo.File(
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.Launcher/Extensions/LoggerConfigurationExtensions.cs:31]: Should we maybe add a `retainedFileCountLimit` of e.g. 60 days?

File: src/fiskaltrust.Launcher/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.Launcher/Properties/launchSettings.json:2]: Do we need this file? If yes, I guess we could cleanup the IIS stuff 😄

**Review by TSchmiedlechner — COMMENTED** [review]
Very impressive work 👍 just added a few small comments, but lgtm

File: src/fiskaltrust.Launcher/Extensions/LoggerConfigurationExtensions.cs (hunk 1 above)
Comment (volllly) [reply src/fiskaltrust.Launcher/Extensions/LoggerConfigurationExtensions.cs:31]: the default `retainedFileCountLimit ` is 31 files and `retainedFileTimeLimit` is none 🤷

**Pull request merged**